libbeat/scripts/Makefile

```diff
@@ -30,6 +30,8 @@ COVERAGE_DIR=${BUILD_DIR}/coverage
 COVERAGE_TOOL=${BEAT_GOPATH}/bin/gotestcover
 COVERAGE_TOOL_REPO=github.com/elastic/beats/vendor/github.com/pierrre/gotestcover
 TESTIFY_TOOL_REPO=github.com/elastic/beats/vendor/github.com/stretchr/testify
+GOX=${BEAT_GOPATH}/bin/gox
+GOX_REPO=github.com/mitchellh/gox
```

Comment:
Do we need this as variable? I recommend to only use variables if we have multiple usages as our Makefile is already quite messy / long.